modules/verifybamid2/main.nf

```diff
@@ -16,11 +15,13 @@ process verifybamid2 {
 
     script:
     def reference = ref_fasta ? /--Reference "${ref_fasta}"/ : ''
+
+    vf_threads = (int) Math.ceil(task.cpus*2)
     """
    # run verifybamid2
    # get the percentage of cross-individual contamination rate 
     
-    verifybamid2 --NumPC 4 --SVDPrefix ${params.vbi2_svdprefix} ${reference} --BamFile ${bam} --Output ${sample}
+    verifybamid2 --NumPC 4 --NumThread ${vf_threads} --SVDPrefix ${params.vbi2_svdprefix} ${reference} --BamFile ${bam} --Output ${sample}
```

Comment:
This should suffice:
```suggestion
    verifybamid2 --NumPC 4 --NumThread ${task.cpus*2} --SVDPrefix ${params.vbi2_svdprefix} ${reference} --BamFile ${bam} --Output ${sample}
```

Reply:
strange! when I test last week it `${task.cpus*2}` wasn't worked then I add the `${vf_threads}`